packages/opentelemetry-node/test/propagator.test.ts

```diff
@@ -1,10 +1,207 @@
+import { defaultTextMapSetter, ROOT_CONTEXT, trace, TraceFlags } from '@opentelemetry/api';
+import { suppressTracing } from '@opentelemetry/core';
+import { Hub, makeMain } from '@sentry/core';
+import { addExtensionMethods, Transaction } from '@sentry/tracing';
+import { TransactionContext } from '@sentry/types';
+
 import { SENTRY_BAGGAGE_HEADER, SENTRY_TRACE_HEADER } from '../src/constants';
 import { SentryPropagator } from '../src/propagator';
+import { SENTRY_SPAN_PROCESSOR_MAP } from '../src/spanprocessor';
+
+beforeAll(() => {
+  addExtensionMethods();
+});
 
 describe('SentryPropagator', () => {
   const propogator = new SentryPropagator();
```

Comment:
this is still a `propogator` leftover ;)

Reply:
dang it!!!

Reply:
fixed with bff9421